tower/tests/balance/main.rs

```diff
@@ -28,7 +29,9 @@ impl Service<Req> for Mock {
 impl tower::load::Load for Mock {
     type Metric = usize;
     fn load(&self) -> Self::Metric {
-        rand::random()
```

Comment:
Why change this from `rand::random()` to the longer, more explicit version? From my reading of the `rand` docs `rand::random` already picks a uniformly distributed random value from the whole range of `usize` values?

Reply:
I've was able to the initial ugly `Uniform::new_inclusive`, but we won't be able to use `rand::random()` because the necessary traits are not implemented on usize anymore.